build/goGet.sh

```diff
@@ -0,0 +1,9 @@
+GO111MODULE=on go get github.com/jfrog/jfrog-cli;
```

Comment:
I suggest adding '#!/bin/bash' at the beginning.

Reply:
@barbelity,
Since the code in this script in meant to be copied and then pasted in the terminal, through the copy buttons in https://jfrog.com/getcli/ I actually think is is best not to add the '#!/bin/bash' at the beginning.
What do you think?

Reply:
In this case you are right 👍

Reply:
What do you mean @barbelity?
What should we add to the script?